bin/dashboard-server

```diff
@@ -2,7 +2,7 @@
 require_relative '../deployment'
 
 def main
-  rerun = "rerun -p '**/*.{rb,ru,yml}' -d'#{deploy_dir('lib')},#{deploy_dir('shared/middleware')},#{pegasus_dir}' --" if CDO.dashboard_enable_pegasus
+  rerun = "rerun -p '**/*.{rb,ru,yml}' -d '#{deploy_dir('lib')},#{deploy_dir('shared/middleware')},#{pegasus_dir}' -i '**/migrations/*.rb' --" if CDO.dashboard_enable_pegasus
```

Comment:
Sorry I missed this comment earlier, Asher -- this change tells rails server not to restart every time a pegasus migration file is changed. This came up because I was initially developing this as a pegasus migration before I ported it to bin/oneoff.